ebpf/cmd/python_dwarfdump/main.go

```diff
@@ -0,0 +1,399 @@
+// based on https://sourcegraph.com/github.com/grafana/ebpf-autoinstrument/-/blob/pkg/goexec/structmembers.go
```

Comment:
getting a 404 on this one.

Can you add a readme for what this program is for ?

Reply:
https://github.com/grafana/beyla/blob/6b46732da73f2f2cb84e41efdc74789509a7fa2b/pkg/internal/goexec/structmembers.go

fixed the link